src/list/FilterSidebar.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+import { Label, SearchInput, CheckboxGroup, IndeterminateCheckbox } from '@sailthru/stui-components';
+import { fontSizeSmall, fontSizeMedium, fontSizeLarge, fontWeightBold, grayTint90 } from '@sailthru/design-tokens';
+
+const FilterContainer = styled.div`
+    margin-top: 1.875em;
+    padding: 1em;
+    height: 100vh;
```

Comment:
nit-pick: Adding width here will make "User management" stay on one line
```suggestion
    height: 100vh;
    width: 160px;
```